docs/resources/connection_okta.md

```diff
@@ -0,0 +1,86 @@
+---
+page_title: "Resource: auth0_connection_okta"
+description: |-
+  This resource configure your Okta Enterprise Connection to allow your users to use their enterprise credentials to login to your app.
+---
+
+# Resource: auth0_connection_okta
+
+This resource configure your Okta Enterprise Connection to allow your users to use their enterprise credentials to login to your app.
+
+## Example Usage
+
+```terraform
+# This is an example of an Okta Workforce connection.
+
+resource "auth0_connection_okta" "okta" {
+  name           = "okta-connection"
+  display_name   = "Okta Workforce Connection"
+  strategy       = "okta"
```

Comment:
```suggestion
  strategy       = "okta"
```

Does the strategy need to be specified?

Reply:
No, strategy is already specified when you use the okta resource.